docs/CORTX-S3 Server Quick Start Guide.md

```diff
@@ -77,9 +77,14 @@ This guide provides a step-by-step walkthrough for getting you CORTX-S3 Server r
      Once you power on your VM, you can verify if selinux and firewall are disabled by using: `$ getenforce` - you'll get a 'disabled' status.
 
 
-9. You'll need to install CORTX Python Utilities. Follow the steps to install [CORTX Python Utilities](https://github.com/Seagate/cortx-utils/blob/main/py-utils/README.md).
+9. You'll need to install Java OpenJDK 8 by running 
```

Comment:
pls change the wording to 'if not already present'

Reply:
also pls provise the command 'rpm -qa | grep java' so that one can check if it exists

Reply:
Hi @nileshgovande , I updated the PR according to your comments. BTW, the `yum install *` will by default checks whether the package exists or not; it won't install anything if the package is already installed or satisfied.
Let me know if I should do any other changes. Thx!